scripts/export_all_opslevel_yml/opslevel_yml_export_for_all_services.py

```diff
@@ -0,0 +1,86 @@
+import requests
+
+OPSLEVEL_API_TOKEN = "2IwIt50kcpHi0uBmFEYRfHHq35gSqqXEZgF2"
```

Comment:
This script should really accept an argument for `OPSLEVEL_API_TOKEN` or at the least have something more obvious like `OPSLEVEL_API_TOKEN = "PUT-MY-TOKEN-HERE"`